extension/cbasenpc_behavior.cpp

```diff
@@ -265,7 +265,7 @@ BEGINQUERYCALLBACK(ShouldAttack, const CKnownEntity *them)
 ENDQUERYCALLBACK()
 
 BEGINQUERYCALLBACK(IsHindrance, CBaseEntity* blocker)
-	CBPUSHENTITY(blocker)
+	CBPUSHENTITY(blocker == IS_ANY_HINDRANCE_POSSIBLE ? nullptr : blocker)
```

Comment:
I feel like we should expose this to plugins somehow, however not mandatory for this PR.